test/test_files/exceptions/copy/null_pk.test

```diff
@@ -21,7 +21,7 @@ Copy exception: Found NULL, which violates the non-null constraint of the primar
 ---- 2
 alice
 bob
--STATEMENT CALL show_warnings() RETURN query_id, message, file_path, skipped_line, line_number
+-STATEMENT CALL show_warnings() RETURN query_id, message, file_path, skipped_line_or_record, line_number
```

Comment:
Should we still keep using the original term `skipped_line`?

Reply:
The thing is that it is possible for it not to be a line (it can be part of or multiple lines) so I feel like `skipped_line` is a bit misleading